### DISCUSSION TITLE
Where to publish papers about mappings?

### DISCUSSION THREAD

**Discussion by saubin78** [mapping-commons/sssom#272] [General]
Hi,
Any suggestions of journals that would be interested in publishing **data papers** to describe mapping sets and show how we use SSSOM?
The mappings sets we want to describe are in the domains of agriculture & ecology.

May thanks in advance for your ideas

**Comment by matentzn** [comment]
Very nice question! I would think that the typical culprits like JBMS, WSJ could still be interesting, but if it was me, I would go for a FAIR data/Open Science/Open Data type journal - unfortunately I don't have any experience with these. Maybe someone else, like @graybeal ?

**Reply by saubin78** [reply]
Indeed,  a FAIR data/Open Science/Open Data type journal would be better.

**Comment by matentzn** [comment]
I asked a bit around and Pascal Hitzler recommended to check out https://cacm.acm.org/ which is very general but reaches a wide audience..

I also looked for myself and I think I like https://www.nature.com/sdata/principles a bit!

**Reply by matentzn** [reply]
Some more suggestions:

https://force11.org/
https://onlinelibrary.wiley.com/journal/17414857

**Comment by saubin78** [comment]
Thanks @matentzn . Would  https://f1000research.com/for-authors/article-guidelines/data-notes be a valuable option?
